components/patient-access-rules/ui/src/main/resources/PhenoTips/PatientAccessRightsManagement.xml

```diff
@@ -94,12 +94,39 @@
   #set ($collaborators = $request.getParameterValues('collaborator'))
   #set ($accessLevels = $request.getParameterValues('accessLevel'))
   #set ($owner = "$!{request.get('owner')}")
+  #if ("$!{owner}" != "")
```

Comment:
It doesn't seem that the case where the ```owner``` is empty is checked.
The ```#elseif ($invalidOwner)``` and ```#elseif ($cannotTransfer)``` below will evaluate to ```false```.
As far as I can read in the code, that will allow changing the collaborators with an empty ```owner```.

The safest thing to do would be to initially assume that the owner is invalid, and ```cannotTransfer``` is true, by setting those variables before the ```if``` clause.